web-client/src/presenter/computeds/formattedMessages.test.js

```diff
@@ -488,4 +491,117 @@ describe('formattedMessages', () => {
 
     expect(result.messages.length).toEqual(2);
   });
+
+  it('should set showTrialInformation to true when caseStatus is calendared', () => {
+    const mockCalendaredMessage = {
+      caseStatus: STATUS_TYPES.calendared,
+      completedAt: '2019-01-02T16:29:13.122Z',
+      createdAt: '2019-01-01T16:29:13.122Z',
+      docketNumber: '101-20',
+      message: 'This is a test message',
+      trialDate: '2025-01-01T16:29:13.122Z',
+      trialLocation: 'Austin, TX',
+    };
+    const result = runCompute(formattedMessages, {
+      state: {
+        messageBoxToDisplay: {
+          box: 'outbox',
+        },
+        messages: [mockCalendaredMessage],
+        screenMetadata: {},
+        user: {
+          role: 'adc',
+        },
+      },
+    });
+
+    expect(result.messages[0].showTrialInformation).toBeTruthy();
+  });
+
+  it('should set showTrialInformation to false when caseStatus is NOT calendared', () => {
+    const mockCalendaredMessage = {
+      caseStatus: STATUS_TYPES.new,
+      completedAt: '2019-01-02T16:29:13.122Z',
+      createdAt: '2019-01-01T16:29:13.122Z',
+      docketNumber: '101-20',
+      message: 'This is a test message',
+    };
+
+    const result = runCompute(formattedMessages, {
+      state: {
+        messageBoxToDisplay: {
+          box: 'outbox',
+        },
+        messages: [mockCalendaredMessage],
+        screenMetadata: {},
+        user: {
+          role: 'adc',
+        },
+      },
+    });
+
+    expect(result.messages[0].showTrialInformation).toBeFalsy();
```

Comment:
❓ Since the test is specifically expecting a boolean, should the expect specifically reflect that? 
`false` vs `falsy`

Reply:
Good idea, on it!